tla-types/src/main/scala/at/forsyte/apalache/tla/typecheck/TypeContext.scala

```diff
@@ -30,16 +30,16 @@ class TypeContext(val largestVarIndex: Int, val types: Map[String, (TlaType1, Se
 object TypeContext {
 
   /**
-   * The value of the variable index that means that all variables can be quantified.
+   * A type scheme that carries a type and a set of variables that are universally quantified.
```

Comment:
What exactly is the relationship between between the `TlaType1` and the set of variables? Is it that the set of variables are all bound to that type?

Reply:
It is a subset of type variables used in the type. Some of the type variables may be bound in outer scopes, so they should not be quantified.